openrr-client/src/clients/ik_client.rs

```diff
@@ -235,3 +239,87 @@ pub fn check_ik_with_interpolation_and_constraints(
     }
     Ok(traj)
 }
+
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct IkSolverConfig {
+    pub root_node_name: Option<String>,
+    pub ik_target: String,
+    pub use_random_ik: bool,
+    pub allowable_position_error_m: f64,
+    pub allowable_angle_error_rad: f64,
+    pub jacobian_multiplier: f64,
+    pub num_max_try: usize,
+}
+
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct IkClientConfig {
+    pub name: String,
+    pub client_name: String,
```

Comment:
Comment:
As same as before PR, name sounds a little ambiguous for me.
I don't understand whole structure actually, I can't say that what is the best.
(name -> xxx_key ?)